specification/configuration.md

```diff
@@ -1,5 +1,7 @@
 # Configuration
 
+**Status**: [Stable](../README.md)
```

Comment:
Should this link to the specific section?

Reply:
That sounds like a good idea to me. :+1: